settings.gradle

```diff
@@ -1,3 +1,12 @@
+plugins {
+    id 'org.ajoberstar.reckon.settings' version "0.18.3"
+}
 rootProject.name = 'alfresco-sdk'
 
-enableFeaturePreview('STABLE_PUBLISHING')
\ No newline at end of file
+reckon {
+    defaultInferredScope = 'patch'
+    snapshots()
+    scopeCalc = calcScopeFromProp()
```

Comment:
I haven't used reckon myself, but if you use `calcScopeFromProp`, don't you need commit messages that follow the format outlined in https://github.com/ajoberstar/reckon?tab=readme-ov-file#reading-scope-from-commit-messages ? I don't see those in the history of this repo.

Reply:
That is for when you use `calcScopeFromCommitMessages()`, not `calcScopeFromProp()` (which reads from gradle properties that can be passed on the commandline)